install-dev/upgrade/sql/1.7.2.0.sql

```diff
@@ -0,0 +1,3 @@
+
+ALTER TABLE  `PREFIX_stock_available` ADD  `physical_quantity` INT NOT NULL DEFAULT  '0' AFTER  `quantity`;
+ALTER TABLE  `PREFIX_stock_available` ADD  `reserved_quantity` INT NOT NULL DEFAULT  '0' AFTER  `quantity`;
```

Comment:
@thierrymarianne 
I think you have to add an entry for AdminStockManagement tab! (like your xml/tab.xml)